lib/open_mic.rb

```diff
@@ -0,0 +1,18 @@
+class OpenMic
+  attr_reader :location, :date, :performers
+  def initialize(info)
+    @info = info
+    @location = @info[:location]
+    @date = @info[:date]
```

Comment:
this is okay to do here, you could simplify it a bit since we don't really need to recall `@info` anywhere else